bin/hdfcoinc/pycbc_page_segtable

```diff
@@ -108,8 +108,11 @@ for segment_name in opts.segment_names:
 
     # put comment in caption
     caption += " " + segment_name
-    if comment_dict[key] != None:
-        caption += " ("+comment_dict[key]+")"
```

Comment:
Ok Ian, you're right. At one point it was correct. Anyways, I noticed your PR has an older version of ``pycbc_page_segtable``, that code looks quite a bit different on master now. Though the same problem is on master.

I have a branch that fixes the caption issue, here's the fix to the fix: https://github.com/cmbiwer/pycbc/blob/seg_pr/bin/hdfcoinc/pycbc_page_segtable#L120

Basically the couple lines need to be indented.

Gives a table like: https://sugar-jobs.phy.syr.edu/~cbiwer/test.html

Open the table with ``vim`` you'll see the meta-data tag has comments in it. The example XML file is on sugar here: /home/cbiwer/src/pycbc_2/H1L1-FOREGROUND_CENSOR-1127777600-398800.xml

Should I make a new PR or is this something you want to add here?